tests/test_master/test_mixin.py

```diff
@@ -734,7 +734,7 @@ def test_transaction_instance_saved_once_simple_case(mocker):
 
     mapper = (
         (i0.pk, 0, 'old', None),
-        (i1.pk, 0, '2', '1'),
+        (i1.pk, 0, '2', None),
```

Comment:
That proves that your changes work, but it's a one scenario as create and save are done in the same transaction.

```
i1 = models.TrackedFieldsParentModel.objects.create(char_field='1')
        i1.char_field = '2'
        i1.save()
```

Let's also check the one, where we load instance with some value from DB and do double saves if such scenario doesn't exist.

Reply:
👍